NotificationService/NotificationService.Data/Repositories/EmailNotificationRepository.cs

```diff
@@ -36,7 +38,12 @@ public class EmailNotificationRepository : IEmailNotificationRepository
         /// <summary>
         /// Instance of <see cref="Container"/>.
         /// </summary>
-        private readonly Container cosmosContainer;
+        private readonly Container emailHistoryContainer;
```

Comment:
Do we need separate container? can we use same container with diff tables?

Reply:
we do not have tables inside container for cosmos sql API implementation